frontend/lib/api/ApiError.ts

```diff
@@ -56,6 +56,15 @@ export class NotFoundError extends FatalError {
   }
 }
 
+export function isSucceess<T>(result: ApiResult<T>): result is ApiResponse<T> {
```

Comment:
typo:
```suggestion
export function isSuccess<T>(result: ApiResult<T>): result is ApiResponse<T> {
```
(also need to rename references)